test/plugin/customParseFormat.test.js

```diff
@@ -72,3 +75,47 @@ it('fails with an invalid format', () => {
   expect(dayjs(input, format).format().toLowerCase())
     .toBe(moment(input, format).format().toLowerCase())
 })
+
+it('parse month from string', () => {
+  const input = '2018 February 03'
+  const format = 'YYYY MMMM DD'
+  expect(dayjs(input, format).valueOf()).toBe(moment(input, format).valueOf())
+})
+
+it('parse month from short string', () => {
+  const input = '2018 Feb 03'
+  const format = 'YYYY MMM DD'
+  expect(dayjs(input, format).valueOf()).toBe(moment(input, format).valueOf())
+})
+
+it('parse month from short string in other locales', () => {
+  const input = '2018 May 03'
+  const inputUk = '2018 трав 03'
+  const format = 'YYYY MMM DD'
+  dayjs.locale(uk)
+  expect(dayjs(inputUk, format).valueOf()).toBe(moment(input, format).valueOf())
+})
+
+it('return Invalid Date when parse corrupt string', () => {
+  const input = '2018 Februaru 03'
+  const format = 'YYYY MMMM DD'
+  expect(dayjs(input, format).format()).toBe('Invalid Date')
```

Comment:
where's `'Invalid Date'`?

Reply:
What do you mean?

Reply:
As a user I expect to know if something goes wrong when parsing my date string.
In first iteration I raised Error in parsing function in result was "Invalid Date". But now it returns correct date (maybe it previous value). So I consider to bring back errors raising.

Reply:
Take a look at test details
https://travis-ci.org/iamkun/dayjs/jobs/495012516#L543

Reply:
May be you are right, let's throw this error 😬

Reply:
Looks like this done at last :)